docs/src/main/sphinx/connector/redis.rst

```diff
@@ -140,6 +141,16 @@ files, which must end with ``.json`` and contain table description files.
 
 This property is optional; the default is ``etc/redis``.
 
+``redis.table-description-cache-ttl``
```

Comment:
The property needs more explanation. I suppose existing users deploy same table description files to all nodes. They will under what happens if those files are different among nodes.